sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java

```diff
@@ -0,0 +1,125 @@
+import com.azure.resourcemanager.mysql.models.ServerPropertiesForDefaultCreate;
+import com.azure.resourcemanager.mysql.models.ServerVersion;
+import com.azure.resourcemanager.mysql.models.Sku;
+import com.azure.resourcemanager.mysql.models.SkuTier;
+import com.azure.resourcemanager.mysql.models.SslEnforcementEnum;
+import com.azure.resourcemanager.mysql.models.StorageProfile;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class MySqlManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.EUROPE_WEST;
+    private String resourceGroupName = "rg" + randomPadding();
+    private MySqlManager mysqlManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        mysqlManager = MySqlManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateServer() {
+        Server server = null;
+        String randomPadding = randomPadding();
+        try {
+            String serverName = "mysql" + randomPadding;
+            String adminName = "sqlsa" + randomPadding;
+            // @embedmeStart
+            server = mysqlManager.servers()
+                .define(serverName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withProperties(
+                    new ServerPropertiesForDefaultCreate()
+                        .withAdministratorLogin(adminName)
+                        .withAdministratorLoginPassword("!QA2ws#ED4rf")
```

Comment:
Add a comment that user MUST change the password.

Reply:
The password setting has been changed to randomly generated.

Reply:
OK. Make sure your local run still pass.